sunpy/net/dataretriever/tests/test_kanzelhohe.py

```diff
@@ -46,7 +46,7 @@ def test_query():
 
 @pytest.mark.online
 @pytest.mark.parametrize("time, wavelength",
-[(Time('2015/01/02 07:30:00', '2015/01/02 07:38:00'), Wavelength(32768*u.AA))])
+[(Time('2015/01/02 07:30:00', '2015/01/02 07:38:00'), Wavelength(3276.8*u.nm))])
 #This test downloads 3 files
 #Each file is 4.5MB, total size
 #is 13.4MB
```

Comment:
I think we will get into troubles downloading so much for each client... we will have to figure out a different way of testing all the clients... otherwise travis will time out and people will hate us each time that run all the tests.